### DISCUSSION TITLE
Release arm64 versions of the CLI

### DISCUSSION THREAD

**Discussion by jomifepe** [bitwarden/clients#8221] [Password Manager]
### Code Contribution Proposal

Hello, I see that x64 CLI binaries are released on GitHub, but not an arm64. Is there a reason?

Currently, the only way to install an arm64 version of the CLI on macOS is using homebrew or npm (as stated in the [official install guide](https://bitwarden.com/help/cli/#download-and-install)). This is far from ideal, as those are external tools, and the x64 versions are there, available for download, so why not also have the arm64 ones? Also, in Apple's case, ARM is the new standard, and people with relatively new macs (not that new, as M1 was released back in 2020) will use the arm64 version.

That said, there have been efforts to start releasing arm64 binaries, and I was wondering if this could get some attention.

1. https://github.com/bitwarden/clients/pull/2976
2. https://github.com/bitwarden/clients/pull/7338

Related forum posts: 
- https://community.bitwarden.com/t/arm-support-for-cli/42362
- https://community.bitwarden.com/t/release-macos-arm64-version-of-cli/63985

Thank you.

**Comment by jomifepe** [comment]
Closed as outdated, but what's outdated about this issue? Is it because of the _new_ `bitwarden-cli-*-npm-build` artifact?

**Comment by dwbit** [comment]
Hey @jomifepe, Github discussion is used for discussing code contributions you wish to write, is this something you're interested in?

**Reply by jomifepe** [reply]
Ah, makes sense 🙂 Yes, I am interested in contributing, but for that I created [a more recent discussion](https://github.com/orgs/bitwarden/discussions/10219).
